igor_src/Aligner.cpp

```diff
@@ -251,6 +251,93 @@ forward_list<Alignment_data> Aligner::align_seq(string nt_seq , double score_thr
 	return alignment_list;
 }
 
+
```

Comment:
For ease of maintenance and for the sake of documentation we should try and write the doxygen docs for each new function we introduce, it is much easier to do on the fly than having to write the whole doc a posteriori